constitution.md

```diff
@@ -0,0 +1,196 @@
+Membership will be revoked by ½ vote of officers plus ¾ vote from the general
+membership if actions are deemed inappropriate by the membership.
+
+## Article IV - Officers
+
+Election of officers will require a majority vote from the general membership.
+If a candidate fails to receive a majority of votes, a run off election will be
+held within the top two candidates that received the most votes. Members
+interested in becoming an officer must meet the following academic requirement:
+2.2 cumulative G.P.A. The term of office will be one full year (when to when).
+All officers shall comprise the Executive Committee of the organization. The
+Executive Committee shall meet in addition to regular organization meetings.
+The Executive Committee shall appoint such committees that are needed to carry
+out organization goals. The officers and individual duties shall be:
+
+- President
+  - Preside over all meetings
+  - Represent organization on campus
+  - Ensure that the organization is operating in conformity with the standards
+  set forth by James Madison University and the Office of Student Activities &
+  Involvement
+  - Cosign organization checks with Treasurer
+  - Maintain communication with organization adviser
+
+- Vice-president
+  - Preside over meetings in the absence of the President
+  - Schedule meetings/events with appropriate University offices
+  - Coordinate organization promotion and publicity of events
+
+- Secretary
+  - Maintain an accurate record of all organization meetings and post for
+members
+  - Maintain membership directory
+  - Correspond when necessary with University administration and other
+recognized organizations
+
+- Treasurer
+  - Maintain accurate record of organization transactions
+  - Collect dues if required
+  - Develop organization budget and present to membership for ¾ vote
+  - Cosign organization checks with President
+  - Arrange fundraising opportunities for the organization
+  - Solicits additional funding if needed from the Student Government
+Association in conjunction with the President
+
+Officers may be removed from office by ½ vote of the other officers and ¾ of
+the general membership if actions are deemed inappropriate by the membership.
+The officer is permitted to speak before the Executive Committee and the
+general membership about the charges made concerning his/her performance. The
+officer is not permitted to participate in the deliberation of the Executive
+Committee regarding the charges.
+
+## Article V - Adviser
+
+The adviser shall be a full or part time faculty or staff member at James
+Madison University. The adviser will assume those responsibilities as outlined
+in this constitution and/or found in University Policy No. 3101 entitled
+"Faculty Advisers to Student Organizations." The adviser will be selected by a
+½ vote of the Executive Committee and then presented to the general membership
+for a ¾ vote. Advisers not fulfilling responsibilities or abiding by the
```

Comment:
Though not part of the standard template, I would like to see a clause that advisor removal cannot begin without a tentative successor chosen by the exec. There should not be a break in advisor continuity.